app/assets/stylesheets/events.css.scss

```diff
@@ -213,6 +213,17 @@ article.type-system-geometric {
   }
 }
 
+#thankYous {
```

Comment:
Avoid using id selectors<br>Selector `thankYous` should be written in lowercase with hyphens